doc_source/cni-upgrades.md

```diff
@@ -63,4 +65,75 @@ Any changes you've made to the plugin's default settings on your cluster can be
 
       ```
       kubectl apply -f aws-k8s-cni.yaml
-      ```
\ No newline at end of file
+      ```
+**Custom upgrade scenarios when CNI custom networking in place**
+
+The upgrade might be an involved process when custom networking is in place. While there are efforts underway to make this process seamless, the best practice is to consider backup, analyze, and update resources to minimize undesirable application downtime\. It is important to understand the exact steps might vary across different versions\. Below steps work best when only an image change is required.
+
+***Backup***
+  + Take a backup of current CNI daemonset\.
+
+    ```
+    kubectl get daemonset aws-node -n kube-system -o yaml > aws-k8s-cni-old.yaml
+    ```
+  + Download the manifest file\.
+
+    ```
+    curl -o aws-k8s-cni.yaml https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.7.5/config/v1.7/aws-k8s-cni.yaml
+    ``` 
+  + Compare the ENV differences, image version changes\.
+
+***Update***
+
+  + Inplace edit
+    + Edit manifest\.
+      ```
+      kubectl edit daemonset aws-node -n kube-system 
+      ```
+    + Change the container image version and save\.
```

Comment:
Also, it will be good to add a warning to check release notes since some upgrades would need to apply the entire manifest or atleast the newly introduced dependency.